### PR TITLE
*: Change GitHub issue template to use the forum

### PR DESCRIPTION
**Pull request by dveeden**
### What problem does this PR solve?

Try to move question to the forum while keeping bugs on GitHub

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
```
Comment (winkyao) [inline .github/ISSUE_TEMPLATE/config.yml:1]: >If you set blank_issues_enabled to true, people will have the option to open a blank issue.

we should encourage the contributors to use issue template?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+contact_links:
+  - name: Ask PingCAP
```
Comment (winkyao) [inline .github/ISSUE_TEMPLATE/config.yml:3]: I think name it "TiDB Forum" is better?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    url: https://ask.pingcap.com/
+    about: Ask a question on the TiDB Forum
+  - name: Ask TUG
```
Comment (winkyao) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion
  - name: TiDB Forum(Chinese)
```

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (dveeden) [reply .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name: TiDB Forum
```

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (dveeden) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Yes, but I think there is no reason to restrict this now. I'm ok with any setting here.

**Review by winkyao — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (xhebox) [reply .github/ISSUE_TEMPLATE/config.yml:1]: I think `blank_issues_enabled: false` is more reasonable

**Review by xhebox — COMMENTED** [review]
Rest LGTM

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (dveeden) [reply .github/ISSUE_TEMPLATE/config.yml:1]:
```suggestion
blank_issues_enabled: false
```

**Review by xhebox — APPROVED** [review]

**Pull request merged**